src/stores/A_store.js

```diff
@@ -60,7 +60,7 @@ class A_store extends ReduceStore {
             }
 
             case ActionTypes.RESET: {
-                return {pathGrid: new Grid(SIZE, SIZE), player: {x: -1, y: -1}, end: {x: -2, y: -2}, stage: 'STARTP', path: ''};
+                return {pathGrid: new Grid(SIZE, SIZE), player: {x: -1, y: -1}, end: {x: -2, y: -2}, stage: 'STARTP', path: '', visibleCells: '', seenCells: ''};
```

Comment:
feeling like this should have newlines now <img src="https://cdn.discordapp.com/emojis/361918777262080004.png" alt=":blobcatdizzy:" width="22" height="22">

also, path and visibleCells and seenCells initialized to a string?

Reply:
🐇